source/apiVolontaria/volunteer/resources.py

```diff
@@ -16,7 +16,7 @@ class ParticipationResource(resources.ModelResource):
     phone = Field()
     mobile = Field()
     cell = Field()
-    task_type = Field(column_name=_('task_type'))
+    task_type = Field(column_name='task_type')
```

Comment:
Since the column name is the readable column by the final user it's important to keep it I18N. Why do you removed the I18N function here ?

Reply:
@RignonNoel Not sure why, but I've place it back !